modules/retail/pages/retail-offline.adoc

```diff
@@ -0,0 +1,42 @@
+[[retail.offline]]
+= Offline Use
+
+If the {susemgr} Server is offline, you can still perform a reboot on the terminals.
+If the connection between the branch server and the terminals is unstable or has low bandwidth, you can still perform an update on the terminals, using caching.
+
+[IMPORTANT]
+====
+Offline features are available only for {susemgr} Server 4.0 and above.
+You must have the Saltboot and PXE formula packages installed.
+====
+
+== Offline Terminal Reboot
+
+If the {susemgr} Server is offline, and a terminal is rebooted, it will fall back to a previously installed image.
+
+This will occur in these situations:
+* If the Saltboot formula has not started within a specified time (default value is 60 seconds).
+* If the terminal does not acknowledge that the Saltboot formula has started.
+* If the root partition is specified on the kernel command line (handled by the PXE formula), is mountable (and is not encrypted), and contains [path]``/etc/ImageVersion`` (which is created by Kiwi).
+
+The timeout configuration option is in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+The parameter is measured in seconds, and defaults to [systemitem]``60``.
+
+----
+SALT_TIMEOUT = 60
+----
+
+When you set this parameter, it will come into effect only when packages are synchronized.
+You might like to delete the cached data to remove older data.
+
```

Comment:
This is timeout checked on terminals, it is unrelated to server configuration in /etc/rhn/rhn.conf and unrelated to packages.

The relevant details are here:
https://github.com/SUSE/doc-susemanager/blob/master/modules/retail/pages/retail-formula-pxe.adoc#kernel-command-line-parameters